health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/Inbox.js

```
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Inbox = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)